lib/rules/index.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   'no-abstract-roles': require('./lint-no-abstract-roles'),
   'no-action-modifiers': require('./lint-no-action-modifiers'),
   'no-attrs-in-components': require('./lint-no-attrs-in-components'),
+  'no-args': require('./lint-no-args'),
```

Comment:
Should this be `no-args-paths` instead? We actually _do_ allow `this.args` so saying `no-args` seems a tad bit off.